code/game/machinery/doors/door.dm

```diff
@@ -222,6 +222,10 @@
 /obj/machinery/door/proc/try_to_crowbar(obj/item/I, mob/user)
 	return
 
+/obj/machinery/door/proc/try_to_crowbar_secondary(obj/item/I, mob/user)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
Also, document it.